sdk/identity/Azure.Identity/src/Credentials/TokenCredentialOptions.cs

```diff
@@ -32,11 +32,11 @@ public Uri AuthorityHost
         }
 
         /// <summary>
-        /// Gets or sets value indicating if ETW logging that contains PII content should be logged.
-        /// Setting this property will not disable redaction of <see cref="Request"/> Content. To enable logging of sensitive <see cref="Request.Content"/>
+        /// Gets or sets value indicating if ETW logging that contains potentially sensitive content should be logged.
+        /// Setting this property to true will not disable redaction of <see cref="Request"/> Content. To enable logging of sensitive <see cref="Request.Content"/>
```

Comment:
Should we add the same blurb from the changelog here?

> This equates to passing 'true' for the `enablePiiLogging` parameter to the 'WithLogging' method on the MSAL client builder.

Reply:
Added.